tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/alerts/alerts_component.css

```diff
@@ -15,13 +15,24 @@ limitations under the License.
 
 .alerts-breakdown-container {
   font-size: 13px;
-  padding: 10px 10px 10px 50px;
+  padding: 10px 10px 10px;
   position: relative;
 }
 
+.alerts-breakdown-type {
+  border-radius: 0px 10px 10px 0px;
+  cursor: pointer;
+  line-height: 30px;
```

Comment:
if possible, avoid setting explicit line-height at all cost. I have seen so many screwed up looking UI (e.g., What If Tool's settings page) with that. Instead, use flex container and set an explicit height if that's what you mean.

Reply:
`line-height` is needed here to make sure that the children of this element are placed vertically in the middle.

Reply:
flex-container should solve that with ease. I will not block LGTM even if it can be converted.

Reply:
Done. Thanks.